### ISSUE TITLE
USED % is not getting to zero after everyrun in selenoid UI

### ISSUE THREAD

**Issue by ankur1233** [aerokube/selenoid#943]
Even I'm not running any browser instance at time in selenoid it's **USED percent is 60%** and in  **QUOTA 0 + 3 / 5**.

Expected:- After every run, it should be **0%** and **QUOTA 0 + 0 / 5**

![image](https://user-images.githubusercontent.com/18676332/85768775-013bc500-b737-11ea-991b-6a295be3a508.png)

**Comment by vania-pooh** [comment]
@ankur1233 you have 3 pending sessions, i.e. starting containers. 3/5 = 60%.

**Comment by ankur1233** [comment]
How it went to pending ??I did not start any session.

**Comment by vania-pooh** [comment]
@ankur1233 could be because sessions are not starting for some reason, but containers still start.

**Comment by ankur1233** [comment]
ok . it's weird don't know how containers are starting automatically. when I use **docker ps** command  I don't see any containers running

**Comment by cheesepaulo** [comment]
I have a similar problem here.
More pending sessions.

Cpu and ram is fine.

![Captura de Tela 2022-08-10 às 3 35 27 PM](https://user-images.githubusercontent.com/13470825/183990567-88954873-6939-43eb-9e54-cd84cab16372.png)
.

**Comment by cheesepaulo** [comment]
Another user reported similar behavior and is related for container be stuck waiting for x server.
https://github.com/aerokube/selenoid/issues/1042

I will disable vnc on selenoid for now, without vnc no need for launch sessions with server I can launch in headless mode.

**Comment by vania-pooh** [comment]
@cheesepaulo could be because of SELinux or similar stuff. We are testing everything on Ubuntu.

**Comment by cheesepaulo** [comment]
After that I started to instantiate the sessions without vnc and the browser in headless mode.

I still needed to go into selenoid and manually kill "docker rm -f" the stuck sessions. I restarted the grid and things started working again.

I did a load test with the empty grid and sent the total capacity of sessions all at once and selenoid was able to go from pending to running in all sessions without major problems.

I use CentOS on this machine. Maybe that's it. CentOS use SELinux by default.

The quick and dirt fix, disable vnc and initialize browsers in headless mode.
The long way, debug SElinux and find a way to avoid this behavior and tell here ❤️
